dashboard/app/models/ability.rb

```diff
@@ -30,7 +30,14 @@ def initialize(user)
         District,
         Workshop,
         Cohort,
-        WorkshopAttendance
+        WorkshopAttendance,
+        # PLC Stuff
+        ProfessionalLearningCourse,
+        ProfessionalLearningModule,
+        ProfessionalLearningTask,
+        PlcEvaluationQuestion,
+        PlcEvaluationAnswer,
+        UserProfessionalLearningCourseEnrollment
```

Comment:
I might suggest that even if ruby lets you name something Plc::Module you shouldn't. :)

> On Feb 16, 2016, at 19:33, Josh Lory <notifications@github.com> wrote:
> 
> In dashboard/app/models/ability.rb:
> 
> > @@ -30,7 +30,14 @@ def initialize(user)
> >          District,
> >          Workshop,
> >          Cohort,
> > -        WorkshopAttendance
> > +        WorkshopAttendance,
> > +        # PLC Stuff
> > +        ProfessionalLearningCourse,
> > +        ProfessionalLearningModule,
> > +        ProfessionalLearningTask,
> > +        PlcEvaluationQuestion,
> > +        PlcEvaluationAnswer,
> > +        UserProfessionalLearningCourseEnrollment
> It might be easier to do the namespacing now? Plc::Module, Plc::EvaluationAnswer, etc. seems way more readable. I was having trouble above knowing which controller I was in b/c the names are so long =/
> 
> —
> Reply to this email directly or view it on GitHub.
>